// Copyright 2018 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

option java_multiple_files = true;
option java_package = "org.chromium.components.sync.protocol";

option optimize_for = LITE_RUNTIME;

package sync_pb;

import "components/sync/protocol/entity_specifics.proto";

// Sync proto to store entity data similar to what the legacy Directory used
// to store, used to persist data locally and never sent through the wire.
//
// Because it's conceptually similar to SyncEntity (actual protocol) and it's
// unclear how big this'll grow, we've kept compatibility with SyncEntity by
// using the same field numbers.
message PersistedEntityData {
  // See corresponding fields in SyncEntity for details.
  optional string name = 8;
  optional EntitySpecifics specifics = 21;
}
